packages/dnb-eufemia/src/extensions/forms/FieldBlock/FieldBlockDocs.ts

```diff
@@ -78,7 +78,7 @@ export const FieldBlockProperties: PropertiesTableProps = {
   },
   composition: {
     doc: 'Use `true` for when you have more than one field wrapped.',
-    type: 'string',
+    type: 'boolean',
```

Comment:
```suggestion
    type: 'true',
```
Not sure if it's correct to say type  `type: 'true'` or not, but TS complains if I do:
`<FieldBlock composition={false}>`:
`Type 'false' is not assignable to type 'true'.ts(2322)`